Sources/PrimerSDK/Classes/User Interface/Root/PrimerRootViewController.swift

```diff
@@ -16,7 +16,7 @@ internal class PrimerRootViewController: PrimerViewController {
     // Child views
     private var backgroundView = PrimerView()
     private var childView: PrimerView = PrimerView()
-    private var nc = PrimerNavigationController()
+    private var nController = PrimerNavigationController()
 
```

Comment:
Nit-pick: If swiftlint is pushing us to have more meaningful names here, I would go the whole hog and use `navigationController` or `navController`. `nController` is not much more informational than `nc` to me.

Reply:
Agree with this

Reply:
oki!